phys/module_microphysics_driver.F

```diff
@@ -745,6 +748,12 @@ smaw3d, &    !Normalized Galperin instability function for momentum  ( 0<= <=4.9
        END IF
        END IF
 
+  !Thomas
+  ! Zero out pwat and lwp before calculation
+  pwat(its:ite,jts:jte)  = 0.
```

Comment:
@thomasedds 
Thomas,
If you have optional fields, you need to test for PRESENT() first.